.github/workflows/docs-publish.yml

```
@@ -0,0 +1,19 @@
+name: Publish documentation to GitHub Wiki
+
+on:
+  push:
+    branches:
+      - main
```

Have this only run on pushes that update the docs directory:
```suggestion
      - main
    paths:
      - 'docs/**'
```